setup.cfg

```diff
@@ -26,6 +26,7 @@ project_urls =
     User Documentation = https://github.com/eprbell/dali-rp2/blob/main/README.md
     Contact = https://eprbell.github.io/eprbell/about.html
 
+# vcpy > 4.4.0 is only compatible with Python 3.10+
```

Comment:
Do we need to drop support for 3.8 and 3.9 in .github/workflows (I'm OK with it)?

Reply:
I guess 3.10 came out late 2020, so it is probably not so unreasonable to ask users to upgrade. It's not such a big issue now that I worked around it, though.

Reply:
I was confused by the comment, which suggests that vrcpy needs Python 3.10 +, so I thought we had to drop support for older versions. Is this correct?

Reply:
Right now, vcr 5.0+ is available, but if we keep it locked to 4.4.0 we can still use Python 3.8, 3.9. If I unlock it, we have to drop support for Python 3.8/9.  That's why the `setup.cfg` file has == and not >=. 

Basically, I really don't need it to be 5.0. It doesn't add that much and this is not mission critical, so I don't need Python 3.10. It works as is. However, if you feel like it is time to drop support I can set it to use the newer versions.  I don't feel strongly either way really. We might have issues with people having problems with not having the right Python version, though.

Anyway, I was waiting for your answer to drop or not to drop before submitting the other changes that is why they are not reflected, yet.

Reply:
Ah, OK, I had misinterpreted the comment. It's OK to fix the version to 4.4 if it allows us to use older Python versions. Later we'll drop 3.8 and 3.9 but it's not urgent right now.